.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+	"typescript.tsdk": "node_modules\\typescript\\lib",
+	"files.eol": "\n",
+	"editor.formatOnSave": true,
```

Comment:
Note to self: what happens when a user does not have the prettier plugin installed and VS Code does the formatting?

Reply:
Good point!
The docs for `typescript.tsdk` says
> Format a file on save. A formatter must be available, the file must not be saved after delay, and the editor must not be shutting down.

So I assume that if there's no formatter installed at all, no formatting is done.

In my latest commit I also added which default formatter to use